webrender/src/renderer.rs

```diff
@@ -1006,59 +1058,59 @@ impl Renderer {
 
         let (data, marker, shader) = match &batch.data {
             &PrimitiveBatchData::CacheImage(ref data) => {
-                let shader = self.ps_cache_image.get(&mut self.device, transform_kind);
+                let shader = self.ps_cache_image.get(&mut self.device, transform_kind).unwrap();
```

Comment:
we could get away without `uwnrap`-ping in each match arm by doing it after the match